database/schema.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE music_album (
+  id INT not null generated always as identity,
+  title VARCHAR(255),
+  published_date DATE
+  on_spotify BOOLEAN,
+  archived BOOLEAN,
+  primary key (id)
+);
+
+CREATE TABLE genre (
+  id INT not null generated always as identity,
+  genre_name VARCHAR(255),
+  music_album_id INT,
+  foreign key (music_album_id) references music_album(id)
+
+CREATE TABLE labels (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  title VARCHAR(255),
+  color VARCHAR(255)
+)
+
+CREATE TABLE books (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  publish_date DATE,
+  publisher VARCHAR(255),
+  cover_state VARCHAR(255)
+)
```

Comment:
- [ ] Please add the `archived`, column, and also add the `label_id`, `author_id`, and `genre_id` as foreign keys to the `books` table while ensuring that they reference the `id` column on the `labels`, `genre`, and `authors` table